docs/user/configuration/integrations/integrations-next/eventhandler-config.md

```diff
@@ -2,17 +2,45 @@
 title = "eventhandler_config"
 +++
 
-# eventhandler_config (beta)
+# eventhandler_config
 
-`eventhandler_config` configures the Kubernetes eventhandler integration. This integration watches [Event](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#event-v1-core) resources in a Kubernetes cluster and forwards them as log entries to a Loki sink.
+`eventhandler_config` configures the Kubernetes eventhandler integration. This
+integration watches
+[Event](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#event-v1-core)
+resources in a Kubernetes cluster and forwards them as log entries to a Loki
+sink. This integration depends on the experimental `integrations-next` feature
+being enabled.
 
-On restart, the integration will look for a cache file (configured using `cache_path`) that stores the last shipped event. This file is optional, and if present, will be used to avoid double-shipping events if Agent or the integration restarts. Kubernetes expires events after 60 minutes, so events older than 60 minutes ago will never be shipped.
+On restart, the integration will look for a cache file (configured using
```

Comment:
Everything below this line I just reformatted to an 80-character line limit so I could read it in my terminal a little more easily